scripts/process-tree.js

```
@@ -0,0 +1,629 @@
+      try {
+        fsApi.rmSync(manifestPath, { force: true });
+        cleanupComplete = !pathExists(manifestPath, fsApi);
+      } catch {
+        // A stale marker is safe to leave behind; its token still prevents a
+        // later run from touching this process tree.
+      }
+    }
+    return { killed };
+  }
+
+  function abortChild(reservation, child, signal = 'SIGKILL') {
```

ESLint

[error] Refactor this function to reduce its Cognitive Complexity from 19 to the 15 allowed.

(sonarjs/cognitive-complexity)